atomic_reactor/plugins/exit_koji_promote.py

```diff
@@ -512,17 +512,22 @@ def get_build(self, metadata):
         if koji_task_id is not None:
             self.log.info("build configuration created by Koji Task ID %s",
```

Comment:
I changed the code below to use that on error if needed. At this point, koji_task_id is expected to be a string since it comes from the build JSON.